src/prebid.js

```diff
@@ -174,6 +174,14 @@ $$PREBID_GLOBAL$$.setTargetingForGPTAsync = function (adUnit) {
   // now set new targeting keys
   targeting.setTargetingForGPT(targetingSet);
 
+  Object.keys(targeting).forEach((key) => {
+    Object.keys(targeting[key]).forEach((key2) => {
+      if (key2 === 'hb_adid') {
```

Comment:
why are we only setting `bid_targeting_set` for winning bid (top bid) when the default in prebid 1.0 is sendAllBids?

Reply:
https://gist.github.com/mkendall07/1ef1b27ed3501d7b03fbd72ec455bab4
A bid picked for targeting (prebid auction win) will be marked bid.status = 'targetingSet' - which indicates it's not eligible for future auctions, until such status is cleared. This will prevent a winning bid to be set in targeting more than 1 time.